screwdriver.yaml

```diff
@@ -11,6 +11,11 @@ jobs:
         steps:
             - vet: go vet ./...
             - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
+            - pretest-setup: |
+                  wget -q -O zstd-cli-linux.tar.gz 'https://bintray.com/screwdrivercd/screwdrivercd/download_file?file_path=zstd-cli-1.4.8-linux.tar.gz'
```

Comment:
once the launcher PR get's merged, won't you have this binary already in place from launcher ? https://github.com/screwdriver-cd/launcher/pull/393/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R98

Reply:
oh just saw your previous comment, so this will change prior to merge ?

Reply:
removed pretest-setup step